Workbooks/Individual Storage/Failures/Failures.workbook

```diff
@@ -37,6 +37,90 @@
       },
       "name": "resource param"
     },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "26332d26-265f-4cef-8b21-8ee048ffd8a8",
+            "version": "KqlParameterItem/1.0",
+            "name": "Logs",
+            "type": 1,
+            "isRequired": true,
+            "query": "(StorageBlobLogs | take 1)\r\n| union (StorageFileLogs | take 1)\r\n| union (StorageQueueLogs | take 1)\r\n| union (StorageTableLogs | take 1)\r\n| summarize c = count()\r\n| project isMissing=iff(c > 0, 0, 1)\r\n| extend isMissing = tostring(isMissing)",
```

Comment:
are all these tables guaranteed to exist?  or can this throw tablenotfound/etc errors?  do you need to protect for that with isfuzzy?  see https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#protect-against-a-table-that-might-not-exist

Reply:
> are all these tables guaranteed to exist? or can this throw tablenotfound/etc errors? do you need to protect for that with isfuzzy? see https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#protect-against-a-table-that-might-not-exist

Yes, the tables are guaranteed to exist; Marcus Heijer verified this.